### ISSUE TITLE
Wrong size for extra data

### ISSUE THREAD

**Issue by muriloloffi** [flathub/com.skype.Client#207]
Edit: I attempted to fix this in PR #208 
Attempt was not successful and I am still having the same issue described below

Original issue text:

On Linux Mint 21.2, the client did not update properly to the latest version - 8.100.0.203 (b9cd6732a4) - through the Update Manager. Then the error shown below is thrown either when trying to update or reinstall using the CLI.

```bash
$ flatpak install com.skype.Client                         
Looking for matches…

com.skype.Client permissions:
    ipc              network                 pulseaudio              x11
    devices          file access [1]         dbus access [2]         system dbus access [3]

    [1] xdg-documents:ro, xdg-download, xdg-music:ro, xdg-pictures:ro, xdg-videos:ro
    [2] com.canonical.AppMenu.Registrar, org.freedesktop.Notifications, org.freedesktop.ScreenSaver, org.freedesktop.secrets,
        org.gnome.GConf, org.gtk.Notifications, org.kde.StatusNotifierWatcher
    [3] org.bluez, org.freedesktop.NetworkManager, org.freedesktop.login1


        ID                       Branch         Op         Remote          Download
 1. [✗] com.skype.Client         stable         i          flathub         126,4 MB / 128,5 MB

Error: Wrong size for extra data https://repo.skype.com/deb/pool/main/s/skypeforlinux/skypeforlinux_8.100.0.203_amd64.deb
error: Failed to install com.skype.Client: Wrong size for extra data https://repo.skype.com/deb/pool/main/s/skypeforlinux/skypeforlinux_8.100.0.203_amd64.deb
```

If the `--verbose` parameter is given, not much additional information is returned for the error.

```
F: Received 126350240 bytes
F: Calling system helper: CancelPull
F: Calling system helper: CancelPull
Error: Wrong size for extra data https://repo.skype.com/deb/pool/main/s/skypeforlinux/skypeforlinux_8.100.0.203_amd64.deb
Updates complete.
error: There were one or more errors
```

**Comment by muriloloffi** [comment]
The @flathubbot made an automatic merge while also mentioning, in the very last comment of the PR #206, a broken link.
@FakeShemp might this be related to the issue I am having?

**Comment by muriloloffi** [comment]
I attempted to fix the issue with the PR #208
